.betterer.results

```diff
@@ -10336,13 +10336,13 @@ exports[`better eslint`] = {
       [216, 21, 3, "Unexpected any. Specify a different type.", "193409811"],
       [240, 21, 3, "Unexpected any. Specify a different type.", "193409811"]
     ],
-    "public/app/plugins/datasource/prometheus/metric_find_query.ts:3246701167": [
+    "public/app/plugins/datasource/prometheus/metric_find_query.ts:1417569751": [
```

Comment:
Not sure what this is? Would you mind explaining the intent here?

Reply:
betterer is a tool to track code quality. the idea is to have fewer `any` types in typescript code. So we would have even stronger typed code. Since my changes interacted with a code that has type `any` it warns me to fix it. But in my case, I cannot focus on that in this PR. So with running `yarn bettered -u` I add those lines to results so it can be ignored for now.
https://github.com/grafana/grafana/blob/main/contribute/create-pull-request.md#betterer